server/controllers/admin/helpdesk.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * The /helpdesk HTTP API endpoint.
+ *
+ * This controller is responsible for implementing READ access for the
+ * help desk info in the Enterprise information.
+ */
+
+const db = require('../../lib/db');
+
+// expose the read function
+exports.read = helpdeskInfo;
+
+/**
+ * @function HelpdeskInfo
+ * @description return the helpdesk info string
+ * Should update to support a passed-in enterprise id
+ */
+
+function helpdeskInfo(req, res, next) {
+
+  const sql = `select helpdesk
+    FROM enterprise
+    LIMIT 1;
+  `;
+
+  db.exec(sql, [])
```

Comment:
If you use `db.one()` this will spit out a JSON object, avoiding the need to do `db.exec(query, params).then(rows => rows[0])`.  `db.one()` is an alias for this operation, returning only the first row of the resulting query.

It also throws an error if any more or less than one row is returned.

Reply:
Done